chall03/rpehkone.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"io/ioutil"
+	"fmt"
+	"strings"
+	"net/http"
+	"strconv"
+)
+
+func main() {
+	res, err := http.Get("https://chall03.hive.fi/")
```

Comment:
You should check `err != nil` after each function that has an error type returned.